README.md

```diff
@@ -29,19 +20,49 @@ go get github.com/ipfs/go-ipld-git
 ```
 
 ## About
+
 This is an IPLD codec which handles git objects. Objects are transformed
-into IPLD graph in the following way:
+into IPLD graph as detailed below. Objects are demonstrated here using both
+[IPLD Schemas](https://ipld.io/docs/schemas/) and example JSON forms.
+
+### Commit
+
+```ipldsch
+type GpgSig string
+
+type PersonInfo struct {
+  date String
```

Comment:
since you're not keeping dates in a human-friendly single string format, I wonder if you could make this unix timestamp an Int too.